runtime/src/runtime/policy.rs

```diff
@@ -77,6 +77,10 @@ pub struct Policy {
 
     pub max_pre_commit_randomness_lookback: ChainEpoch,
 
+    pub prove_commit_ni_sector_batch_max_size: usize,
+    pub max_prove_commit_ni_randomness_lookback: ChainEpoch,
+    pub valid_prove_commit_ni_proof_type: ProofSet,
+
```

Comment:
these need documenting and should be moved below `pre_commit_challenge_delay` so they are consistently placed with the `policy_constants` (and we can match their placement in go-state-types so it's easier to look them up)